src/Microsoft.AspNetCore.WebUtilities/HttpResponseStreamWriter.cs

```diff
@@ -16,11 +16,7 @@ namespace Microsoft.AspNetCore.WebUtilities
     public class HttpResponseStreamWriter : TextWriter
     {
         private const int MinBufferSize = 128;
-
-        /// <summary>
-        /// Default buffer size.
-        /// </summary>
-        public const int DefaultBufferSize = 1024;
+        internal const int DefaultBufferSize = 16 * 1024;
```

Comment:
I would consider making this private unless you think it's important to reference from the InlineData attribute.